tests-legacy/PrestaShopBundle/Controller/ControllerTest.php

```diff
@@ -130,7 +130,6 @@ public function getControllersClasses()
             array('AdminStoresController'),
             array('AdminSuppliersController'),
             array('AdminAttributesGroupsController'),
-            array('AdminEmployeesController'),
```

Comment:
Add the new test in https://github.com/PrestaShop/PrestaShop/blob/develop/tests-legacy/Integration/PrestaShopBundle/Controller/Admin/SurvivalTest.php